cartesian_velocity_controller:
    type: franka_controllers/CartesianVelocityController
    arm_id: panda